src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -83,14 +83,27 @@ class OppgaveService(
                 oppgavetype = Oppgavetype.Fremlegg,
                 beskrivelse = beskrivelse,
                 settBehandlesAvApplikasjon = false,
-                fristFerdigstillelse = iverksett.vedtak.vedtakstidspunkt.toLocalDate().plusYears(1),
+                fristFerdigstillelse = lagFristFerdigstillelseInntektsjekk(iverksett.vedtak.vedtakstidspunkt.toLocalDate()),
                 mappeId = finnMappeForFremleggsoppgave(iverksett.søker.tilhørendeEnhet, iverksett.behandling.behandlingId),
             )
 
         return oppgaveClient.opprettOppgave(opprettOppgaveRequest)?.let { return it }
             ?: error("Kunne ikke finne oppgave for behandlingId=${iverksett.behandling.behandlingId}")
     }
 
+    fun lagFristFerdigstillelseInntektsjekk(vedtaksdato: LocalDate): LocalDate? {
+        // Skal ikke falle på den 6. hver måned
+        // 17. og 18. mai
+        // ikke juli eller august
```

Comment:
Why 😂 

```
SKal ikke falle på den 6. hver måned fordi ...?
Ikke 17. og 18. mai fordi det er forbeholdt to typer utsending av karakterutskrifts-innhententing
Ikke juli eller august fordi ... ? (ferie?) 
```

Siden vi har en såpass magisk sjekk i koden, så er det fint å dokumentere hvorfor dette skjer - så går det an å fjerne denne logikken den dagen de jobber på en annen/bedre måte 🤣

Reply:
hehe, godt poeng, legger det inn i koden. Dobbeltsjekker med Mirja om disse grunnene, men tror de ikke vil at det skal lande på den 6. i måneden fordi inntekt for forrige måned oppdateres i løpet av dagen, slik at det er bedre å vente en ekstra dag for å være sikre på at de har inntekten fra forrige måned.

Reply:
Da er det en finfin begrunnelse å få inn her - for den kunnskapen sitter ikke i veggene 😂